packages/code-du-travail-frontend/src/conventions/texte/Calipso.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Calipso = ({ calipso }) => (
+  <div
+    key={calipso}
```

Comment:
why do you need to add a key here ?

Reply:
this must be a mistake from a previous version, good call